src/mobile/src/ui/components/SeedVaultExportComponent.js

```diff
@@ -298,7 +298,7 @@ class SeedVaultExportComponent extends Component {
         const textColor = { color: theme.body.color };
 
         return (
-            <Animated.View style={[styles.container, { transform: [{ translateX: this.animatedValue }] }]}>
+            <Animated.View style={[styles.container, { transform: [{ withNamespacesX: this.animatedValue }] }]}>
```

Comment:
`translateX` is a prop from https://facebook.github.io/react-native/docs/transforms#transform so it shouldn't be changed